identity/app/views/consentJourney.scala.html

```diff
@@ -147,7 +160,8 @@ <h3 class="manage-account__switch-title">&nbsp;</h3>
 @defaultJourney = @{List(
     marketingConsentStep,
     smsStep,
-    nonElectronicOptOutStep
+    nonElectronicOptOutStep,
```

Comment:
I'm not sure the distinction between non-electronic and profiling to be honest

Reply:
If market research is in non-electronic then maybe profiling should be

Reply:
I did it because the html nesting in otherChannels.scala.html means it's easier to make it a separate module and call it in otherChannels, but yeah your right logically they belong together. But the layout of the page means it's a bit better to separate them.

Reply:
In fact I don't think it makes sense to group them together due to the page layout. I have renamed that module to NonElectronicContact and kept the profiling as a serperate module. That makes the most sense I think.